roles/openshift_version/tasks/main.yml

```diff
@@ -60,13 +60,16 @@
   # It also allows for optional trailing data which:
   # - must start with a dash
   # - may contain numbers
+  # - may containe dots (https://github.com/openshift/openshift-ansible/issues/5192)
+  #
   - name: (Enterprise) Verify openshift_image_tag is valid
     when: openshift.common.deployment_type == 'openshift-enterprise'
     assert:
       that:
-      - "{{ openshift_image_tag|match('(^v\\d+\\.\\d+[\\.\\d+]*(-\\d+)?$)') }}"
+      - "{{ openshift_image_tag|match('(^v\\d+\\.\\d+[\\.\\d+]*(-[\\d+\\.]*)?$)') }}"
```

Comment:
technically this also matches `v1..3` and `v1.2.3...` and such - basically you can add a `.` anywhere.

If we care, regex might be better like:

`(^v\\d+\\.\\d+(\\.\\d+)*(-\\d+(\\.\\d+)*)?$)`